env.example

```diff
@@ -9,4 +9,12 @@ export CLUSTER_NAME="${USER}-aro-cluster"
 export CLUSTER_VNET="${USER}-aro-vnet"
 export ARO_IMAGE=arointsvc.azurecr.io/aro:latest 
 
-. secrets/env
\ No newline at end of file
+# You'll need these to create MIWI clusters with your local RP, but you can comment them
+# out or remove them from your env file if you're only going to be creating service principal
+# clusters.
+export MOCK_MSI_CLIENT_ID="replace_with_value_output_by_hack/devtools/msi.sh"
+export MOCK_MSI_CERT="replace_with_value_output_by_hack/devtools/msi.sh"
+export MOCK_MSI_TENANT_ID="replace_with_value_output_by_hack/devtools/msi.sh"
+export PLATFORM_WORKLOAD_IDENTITY_ROLE_SETS="replace_with_value_output_by_hack/devtools/msi.sh"
```

Comment:
I could see the value in automation to set these env vars, but I consider it out-of-scope for this PR. I snuck in this small, unrelated change here because someone had mentioned to me in Slack that it's difficult to know you need to set these env vars if you miss the corresponding steps [here](https://github.com/Azure/ARO-RP/blob/master/docs/deploy-development-rp.md#prepare-your-environment).